bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/api/dto/clip2/Resource.java

```diff
@@ -121,6 +121,31 @@ public Resource(@Nullable ResourceType resourceType) {
         }
     }
 
+    /**
+     * Check if light or grouped_light resource contains any items.
+     *
+     * @return true is resource doesn't contain any state information
+     */
+    public boolean hasItems() {
```

Comment:
As mentioned before, please rename method as 'hasNonHSBField'

Reply:
Resolving this comment to not have duplicate discussions, I have responded in the other one.